tests/phpunit/tests/option/registration.php

```diff
@@ -149,4 +149,28 @@ public function test_unregister_setting_removes_default() {
 
 		$this->assertFalse( has_filter( 'default_option_test_default', 'filter_default_option' ) );
 	}
+
+	/**
+	 * @ticket 57674
```

Comment:
```suggestion
	/**
	 * The test passes if a Notice | Warning | Error is not raised. Thus. the absence of a Notice | Warning | Error
	 * is an indicator the fix in the ticket resolves the issue.
	 *
	 * @ticket 57674
```

The test name may not be clear enough to understand what the test is doing. Added some context to help.